cmd/ci-secret-bootstrap/secretstats_test.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	coreapi "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/openshift/ci-tools/pkg/testhelper"
+)
+
+func TestGenerateSecretStats(t *testing.T) {
+	testCases := []struct {
+		name                    string
+		secretsByClusterAndName map[string]map[types.NamespacedName]coreapi.Secret
+		expected                secretStats
+		expectedErr             error
```

Comment:
There are no tests which set this, but do we even want to return an error?

The only way a median calculation can "fail" is if there are no values (https://github.com/montanaflynn/stats/blob/master/median.go).  We return `0` in that case (which is fine for logs), so there is never a failure case.

Reply:
Updated.